pomcorn/locators/base_locators.py

```diff
@@ -157,25 +157,51 @@ def __or__(self, other: XPathLocator) -> XPathLocator:
         """
         return XPathLocator(query=f"({self.query} | {other.query})")
 
-    def __getitem__(self, index: int) -> XPathLocator:
-        """Allow get related xpath locator by index.
+    def __getitem__(self, value: int | str | XPathLocator) -> XPathLocator:
+        """Allow to set xpath expressions or index into the locator query.
 
-        Example:
+        Examples:
             div_locator = XPathLocator("//div") --> `//div`
+
+            # Indexation
             div_locator[0]  --> `(//div)[1]`   # xpath numeration starts with 1
             div_locator[-1] --> `(//div)[last()]`
 
+            # Attribute condition
```

Comment:
Let's add 1 example for `str` value and 1 for `XpathLocator`.
Previous examples may be moved to the test